presto-native-execution/presto_cpp/main/QueryContextManager.cpp

```diff
@@ -98,15 +98,30 @@ void updateVeloxConnectorConfigs(
     std::unordered_map<
         std::string,
         std::unordered_map<std::string, std::string>>& connectorConfigStrings) {
+  const auto& systemConfig = SystemConfig::instance();
+
```

Comment:
Do we need to set "connector::hive::HiveConfig::kCacheNoRetentionSession" based on systemConfig->queryDataCacheEnabledDefault() by default before we go to branch below? thanks!